app/src/main/java/org/kiwix/kiwixmobile/main/MainActivity.java

```diff
@@ -1913,23 +1913,14 @@ public void addBooks(List<LibraryNetworkEntity.Book> books) {
     booksAdapter.notifyDataSetChanged();
   }
 
-  private void searchFiles() {
-    if (Build.VERSION.SDK_INT >= VERSION_CODES.M) {
-      if (ContextCompat.checkSelfPermission(this,
+   private void searchFiles() {
+    if (Build.VERSION.SDK_INT >= VERSION_CODES.M&&ContextCompat.checkSelfPermission(this,
```

Comment:
There should be a space before and after `&&`. You should read this https://github.com/kiwix/kiwix-android/blob/master/docs/codestyle.md and autoformat your code using `Ctrl + Shift + L` before every commit to avoid such mistakes.

Reply:
Sorry for that. I assure you I'lll take care of it the next time.